.ruby-version

```diff
@@ -1 +1 @@
-3.3.2
+3.3.4
```

Comment:
Don't update this here. 
Changes like these should have a separate PR on its own as it affects everyone. 
This is the ideal process:
1. File an issue this task.
2. Create a Pull Request
3. Tag everyone (all developers) on that pull request so that everyone can confirm it works for them and they are okay with it. 
4. Once everyone approves, then we can merge it.


These steps are important so as to make sure everyone is comfortable and not a single developer is left-out.

Reply:
Reverting the change to 3.3.2